run_dir/design/worksets.html

```diff
@@ -115,15 +114,14 @@ <h2 class="closed_ws_page_heading"><small class="text-muted">All worksets with p
                 <th class="sort">Closed Projects</th>
               </tr>
           </thead>
+          <tbody class="list" id="closed_ws_table_body"></tbody>
           <tfoot id="closed_ws_table_footer">
               <tr class="darkth">
               <th class="sort">Date Run</th>
               <th class="sort">Workset Name</th>
               <th class="sort">Closed Projects</th>
               </tr>
           </tfoot>
-          <tbody id="closed_ws_table_body">
-        </tbody>
```

Comment:
It should be here. `tfoot` is the placeholder for the search boxes.

Reply:
Or maybe we can leave it there? Does it work when `tbody` is between `thead` and `tfoot`

Reply:
Yes, it works!

Reply:
I had a look and it looks like this
<img width="1789" alt="Screenshot 2024-01-16 at 11 29 02" src="https://github.com/SciLifeLab/genomics-status/assets/11080857/2dba3697-ca8e-4ff6-9208-47c7cbd6d820">

Reply:
Yeah, you're right, I didn't notice the search boxes 👍🏻